### DISCUSSION TITLE
Command to start debug service

### DISCUSSION THREAD

**Discussion by bhunsicker1** [codefori/vscode-ibmi#1118] [Q&A] [answered]
Hello

We are starting to use the debug service on IBM i in Visual Studio Code.  I am wondering
if I can automate the startup of the debug service via a command.   I thought it might be as simple
as adding the DEBUG_SERVICE_KEYSTORE_PASSWORD and DEBUG_SERVICE_KEYSTORE_FILE in the DebugService.env file.
That is not working out for me.  I have noticed when I start the debug service in VSCode, the password
is a different value each time.   
What is the process for using a command to start the debug service?  I would like to add something to lpar startup so that developers do not need to start this.   
What is the minimum security needed to start the debug service?
Thanks

**Comment by worksofliam** [comment]
@bhunsicker1 This is a good question and sadly not something we have control over.

The password that we use is actually just the host name as provided in the connection settings.

Perhaps the IBM docs will guide you further?

https://www.ibm.com/docs/en/merlin/1.0?topic=environment-debugging

Liam

**Comment by bhunsicker1** [comment, marked as answer]
Thank you.

I did change the DEBUG_SERVICE_KEYSTORE_PASSWORD to the host name instead of the password I retrieved from the VSCode log.  
That has gotten me further.  
Thanks
